controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'pry'
+require_relative 'Parser'
+require_relative 'deck'
+require_relative 'view'
+
+class Controller
+  # include Parser
+  attr_reader :viewer, :deck
+
+  def initialize(file)
+    @parsed_file = Parser.read(file)
+    @viewer = View.new
+    @deck = Deck.new(@parsed_file)
+    @deck.generate_random_card
```

Comment:
I think this line of code could be run from inside the Deck's initialize method.